plugins/obs-qsv11/QSV_Encoder_Internal.cpp

```diff
@@ -177,6 +177,23 @@ mfxStatus QSV_Encoder_Internal::Open(qsv_param_t *pParams, enum qsv_codec codec)
 	return sts;
 }
 
+PRAGMA_WARN_PUSH
+PRAGMA_WARN_DEPRECATION
+static inline bool IsOptimizedBRCSupport(const mfxPlatform &platform,
+					 mfxU16 rateControl)
+{
+#if MFX_VERSION_MAJOR >= 2 && MFX_VERSION_MINOR >= 12
```

Comment:
It has occurred to me that should there ever be a libvpl 3.0, this will fail. I'm not sure if that's a scenario we're concerned with, but if it is, this would need to be amended to something like:
```suggestion
#if (MFX_VERSION_MAJOR >= 2 && MFX_VERSION_MINOR >= 12) || MFX_VERSION_MAJOR > 2
```
or, if the construction of `MFX_VERSION` is not expected to change:
```suggestion
#if MFX_VERSION >= 2012
```